ui/app/AppLayouts/Profile/controls/InformationTile.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Rectangle {
+    id: root
+
+    property alias primaryText: primaryText.text
+    property alias secondaryText: secondaryText.text
+    property alias tagsModel: tags.model
+    property alias tagsDelegate: tags.delegate
+
+    height: 52
+    width: layout.width + Style.current.xlPadding
```

Comment:
This should be `implicitWidth` (as designed) and the layout should just fill `root`; same for height (make it `implicitHeight`)